prime-angular-frontend/src/app/modules/enrolment/enrolment.module.ts

```diff
@@ -75,6 +75,10 @@ import { RemoteAccessAddressesComponent } from './pages/remote-access-addresses/
     ContactInformationFormComponent,
     RemoteAccessAddressesComponent
   ],
+  exports: [
+    EnrolleePageComponent,
```

Comment:
Feels like we should reduce the dependencies across modules and use base components directly, or create composite components from base components that are module specific.
* `PageComponent` could be used directly in the PHSA module since it is a shared component, and has only two input bindings, and  
* `ProgressIndicatorComponent` could be a composite component named for the module (ie. `PhsaProgressIndicatorComponent`) since the `EnrolleeProgressIndicatorComponent` uses module specific routes, and we'll want PHSA to use their own routes to determine progression through their enrolment

Reply:
Fixed as suggested